docs/pages/zkvm/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
     "cli-quick-start": "CLI Quick Start",
     "sdk-quick-start": "SDK Quick Start",
-    "configuring-prover": "Configuring the Prover"
+    "configuring-prover": "Configuring the Prover",
+    "sdk-benchmarking": "SDK Benchmarking"
```

Comment:
I would remove "SDK" from the title. It can be inside the document, but I think it's unnecessary from a title perspective.

Reply:
Updated in [36f79a4](https://github.com/nexus-xyz/nexus-zkvm/pull/271/commits/36f79a4ffc509fe44f4c288746c674652d698a4b)